### ISSUE TITLE
Restructure the SimCardio page 

### ISSUE THREAD

**Issue by CZHU20** [SimVascular/simvascular.github.io-archive#115]
Now that svFSI has its own dedicated pages, the existing SimCardio page can be restructured to better demonstrate the scope of the project. I would like to propose a change like this:
![image](https://user-images.githubusercontent.com/43864368/178391977-44856439-e576-4d12-9211-e9d89925f9d7.png)

Here, `Cardiac Geometric Modeling Tools` describes Fanwei's machine-learning based cardiac modeling workflow. `Multiphysics finite element solver` will direct readers to svFSI page. `Purkinje Network Plugin` showcases the generation of Purkinje network using  SimVascular plugin.

**Comment by ktbolt** [comment]
@CZHU20 I think it is better to use `Cardiac Modeling` rather than `SimCardio` to refer to the cardiac modeling functionality. `SimCardio` implies that this is a software package or a tool that is part of SV.

At some point I will replace the Clinical Studies page with an Applications page and move the SimCardio docs there.

**Comment by CZHU20** [comment]
@ktbolt Thanks for the feedback. I can change that to `Cardiac Modeling`.  I am still a bit confused about the role of `SimCardio`. Do we need to mention SimCardio at all on this page?

I think we used to refer to it as "the SimVascular SimCardio software suite" to treat it more like a plugin in SimVascular.

**Comment by ktbolt** [comment]
@CZHU20 I don't think we need to mention SimCardio. SimCardio used to be a repository that contained the Purkinje plugin but that was renamed to SimVascular-Plugin-Purkinje-Network, SimCardio is no longer used and should be removed.

**Comment by CZHU20** [comment]
I see. Will make the changes, but will keep them in my own branch for now.  I have just submitted a revision to the JOSS paper, in which the reviewer asked about the relation between SimCardio and svFSI. I think it's best we postpone the changes to the SimCardio page so as not to confuse the reviewer.

**Comment by CZHU20** [comment]
@ktbolt 

Hi Dave,

Sid’s documentation requires readers to have access to some examples currently hosted on the private repo, mostly contents in these two folders:
https://github.com/StanfordCBCL/Cardiac_Modeling_Tools/tree/master/cardiac-multiphysics-pipeline/01-fibers/biventricular
https://github.com/StanfordCBCL/Cardiac_Modeling_Tools/tree/master/cardiac-multiphysics-pipeline/01-fibers/idealized_lv

Can we host them somewhere with public access?
